libs/datamodel/core/src/transform/ast_to_dml/lift.rs

```diff
@@ -219,6 +219,28 @@ impl<'a> LiftAstToDml<'a> {
                     .filter(|dir| dir.name.name.starts_with(&prefix))
                     .collect_vec();
 
+                let type_specifications_with_invalid_datasource_name = ast_field
+                    .directives
+                    .iter()
+                    .filter(|dir| dir.name.name.contains(".") && !dir.name.name.starts_with(&prefix))
+                    .collect_vec();
+
+                if type_specifications_with_invalid_datasource_name.len() > 0 {
+                    let incorrect_type_specification =
+                        type_specifications_with_invalid_datasource_name.first().unwrap();
+                    let mut type_specification_name_split = incorrect_type_specification.name.name.split(".");
```

Comment:
```suggestion
                    let type_specification_name_split = incorrect_type_specification.name.name.split(".");
```

a small error, need to update